### PR TITLE
Fix fulcrum technician bug

### PR DESCRIPTION
**Pull request by JoeyL6**
Modified the get_postgre_record function to only get the most recent record from postgreSQL if two records have the same fulcrum_id.

### CODE REVIEW

File: transportation-data-publishing/data_tracker/signal_pms_fulcrum.py [hunk 1]
```diff
@@ -250,6 +258,13 @@ def replace_pm_records(
         postgre_records_df["fulcrum_id"].isin(knack_pm_records_df["FULCRUM_ID"])
     ].copy()
 
+    # # get the current year
```
Comment (mateoclarke) [inline transportation-data-publishing/data_tracker/signal_pms_fulcrum.py]: Should we drop this commented out code? Or will you need it in the future?

File: transportation-data-publishing/data_tracker/signal_pms_fulcrum.py [hunk 2]
```diff
@@ -495,4 +511,20 @@ def main():
 
 if __name__ == "__main__":
 
-    print(main())
+
+    # args = cli_args()
```
Comment (mateoclarke) [inline transportation-data-publishing/data_tracker/signal_pms_fulcrum.py]: same thing for this commented out code.

File: transportation-data-publishing/data_tracker/signal_pms_fulcrum.py [hunk 3]
```diff
@@ -42,6 +42,12 @@ def get_postgre_records():
     params = {}
 
     postgre_records = pgrest.select(params=params)
+    postgre_records_df = pd.DataFrame.from_dict(postgre_records)
+
+    # temporary fix to remove duplicate pm records
```
Comment (mateoclarke) [inline transportation-data-publishing/data_tracker/signal_pms_fulcrum.py]: I like having some comments for explanation. I'm wondering, what would be the more permanent fix? Would that be worth adding as a comment?

File: transportation-data-publishing/data_tracker/signal_pms_fulcrum.py [hunk 4]
```diff
@@ -495,4 +511,20 @@ def main():
 
 if __name__ == "__main__":
 
-    print(main())
+
+    # args = cli_args()
+    # app_name = args.app_name
+    app_name = "data_tracker_prod"
```
Comment (mateoclarke) [inline transportation-data-publishing/data_tracker/signal_pms_fulcrum.py]: why the switch to hardcoded string instead of from args?

Comment (JoeyL6) [reply transportation-data-publishing/data_tracker/signal_pms_fulcrum.py]: Hi, Mateo, everything under name == main is for testing only. The script launcher will not read it. But I guess I should comment everything out under name==main to increase some readability.

File: transportation-data-publishing/data_tracker/signal_pms_fulcrum.py (hunk 3 above)
Comment (JoeyL6) [reply transportation-data-publishing/data_tracker/signal_pms_fulcrum.py]: If we are going to use fulcrum for an extended time. I'm going to rewrite some part of this script. Currently, this script blends dictionaries/dataframe operations together which makes it hard to maintain.

File: transportation-data-publishing/data_tracker/signal_pms_fulcrum.py (hunk 1 above)
Comment (JoeyL6) [reply transportation-data-publishing/data_tracker/signal_pms_fulcrum.py]: Good catch, I've deleted that part, thanks!

**Pull request merged**